src/mixins/raster-draw-mixin.js

```diff
@@ -235,7 +235,7 @@ export function rasterDrawMixin(chart) {
                 LatLonUtils.conv900913To4326(pos, pos)
                 const meters = shape.radius * 1000
                 filterObj.shapeFilters.push(
-                  `DISTANCE_IN_METERS(${pos[0]}, ${pos[1]}, ${px}, ${py}) < ${meters}`
+                  `ST_DWithin(CAST(ST_SetSRID(ST_Point(${pos[0]}, ${pos[1]}), 4326) as GEOGRAPHY), CAST(ST_SetSRID(ST_Point(${px}, ${py}), 4326) as GEOGRAPHY), ${meters})`
```

Comment:
NOTE: this always uses an ST_Point constructor, even if the input column is already a POINT type. This is because the px/py is already separated out to be ST_X(point), ST_Y(point) respectively by the time we reach here. To avoid the unnecessary cast would require unwrangling a bunch of upstream code that looks like it could be a bit of a pain, so I decided to punt that detail.